src/views/ops/Adhoc/components/ConfirmRunAssetsDialog.vue

```diff
@@ -0,0 +1,158 @@
+              v-for="asset in runnableAssets"
+              :key="asset.id"
+              :label="asset.name"
+              class="asset-item"
+            >
+              <div class="asset-item">
+                <span>{{ asset.name }}</span>
+                <span class="asset-ip">{{ asset.ip }}</span>
+              </div>
+            </el-checkbox>
+          </el-checkbox-group>
+        </div>
+      </div>
+      <div class="asset-group">
+        <div class="group-title">不可运行资产：</div>
+        <el-checkbox-group class="group-assets">
+          <el-checkbox
+            v-for="asset in failedAssets"
+            :key="asset.id"
+            :label="asset.name"
+            class="asset-item"
+            disabled
+          >
+            <div class="asset-item">
+              <span>{{ asset.name }}</span>
+              <span class="asset-status">{{ asset.error }}</span>
+            </div>
+
+          </el-checkbox>
+        </el-checkbox-group>
+      </div>
+    </div>
+    <div>
+      <div class="selected-count">已选 {{ selectedAssets.length }} 个资产</div>
+    </div>
+  </Dialog>
+</template>
+
+<script>
+import Dialog from '@/components/Dialog'
+
+export default {
+  name: 'ConfirmRunAssetsDialog',
+  components: {
+    Dialog
+  },
+  props: {
+    visible: {
+      type: Boolean,
+      default: false
+    },
+    assets: {
+      type: Object,
+      default: () => {
+      }
+    }
+  },
+  data() {
+    return {
+      searchQuery: '',
+      selectedAssets: []
+    }
+  },
+  computed: {
+    runnableAssets() {
+      return this.assets.runnable
+    },
+    failedAssets() {
+      return [...this.assets.skipped, ...this.assets.error]
+    }
+  },
+  methods: {
+    onCancel() {
+      this.$emit('update:visible', false)
+    },
+    onConfirm() {
+      this.$emit('confirm', this.selectedAssets)
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.confirm-run-assets-dialog {
+
+  .assets-list {
+    max-height: 300px;
+    overflow-y: auto;
+  }
+
+  .asset-group {
+    margin-bottom: 16px;
+
+    .group-title {
+      font-weight: bold;
+      margin-bottom: 8px;
+    }
+
+    .group-assets {
+      ::v-deep .el-checkbox__label {
+        display: inline-block;
+        padding-left: 10px;
+        line-height: 19px;
+        font-size: 13px;
+        width: 100%;
+      }
+
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      grid-row-gap: 5px;
+      justify-items: start;
+
+      .asset-item {
+        display: flex;
+        grid-template-columns: 1fr 1fr;
+        justify-content: space-between;
+        width: 100%;
+
+        .asset-ip {
+          padding-right: 10px;
+        }
+
+        .asset-status {
+          padding-right: 10px;
+          color: #ed5565
+        }
+      }
+
+    }
+  }
+
+  .footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  .selected-count {
+    color: #1ab394;
+  }
+}
+</style>
```

Comment:
There doesn't seem to be any significant code changes between the initial version of the template provided and its current state. However, I'll note that there's an odd spacing issue where two elements with `class="asset-item"` have different line heights when they shouldn't; you may want to fix this minor inconsistency:

**Code Issues:**

1. The `.el-checkbox__label` style is incorrect for one element but not another in `.group-assets`.

2. There's a small discrepancy on how some elements are displayed across the page.

3. In the `.footer`, ensure all items align properly if necessary (margin left should match). The footer needs to maintain a consistent space before and after each item.

4. Fix any other visual inconsistencies which might make the layout less appealing.